fractal.js

```diff
@@ -21,6 +21,25 @@ fractal.components.set('path', src + '/components');
 fractal.components.set('default.preview', '@base');
 
 fractal.docs.set('path', src + '/docs');
+fractal.docs.set('default.status', 'alpha');
+fractal.docs.set('statuses', {
+    alpha: {
+        label: "Alpha",
+        description: "",
```

Comment:
Is there a reason we don't have descriptions yet? Even a simple one would be helpful.